scripts/ci/tools/free-space.py

```diff
@@ -0,0 +1,43 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""free-space.py for cean environment before to start CI"""
+
+import subprocess
+
+exit_code = subprocess.call('./../libraries/_script_init.sh', shell=True)
+print(exit_code)
+
+class style():
+    BLUE = '\033[34m'
+
+print(style.BLUE + "Disable swap")
+subprocess.call(['sudo' , 'swapoff' , '-a'])
```

Comment:
I think it's worth displaying the command we're running. For that, we should add a little helper.
```
def run_command(cmd: List[str], *, check: bool = True, **kwargs):
    print(f"$ {' '.join(shlex.quote(c) for c in cmd)}")
    try:
        subprocess.run(cmd, check=check, **kwargs)
    except subprocess.CalledProcessError as ex:
        print("========================= OUTPUT start ============================")
        print(ex.stderr)
        print(ex.stdout)
        print("========================= OUTPUT end ============================")
    raise
```

Reply:
I am using run_command, and I added a validation for verbose, if verbose is true so print the command